### DISCUSSION TITLE
Amazing plugin, would be perfect for me if

### DISCUSSION THREAD

**Discussion by idanangel** [remotely-save/remotely-save#109] [Ideas]
local changes (new file, delete file, edit file) would be pushed as soon as they happen to the target, and remote changes would be pulled periodically (as it is currently implemented). The issue is mostly with mobile changes, I'd change something and then go to a different app, the home screen, or lock the phone, and the change would be synced to the target only when I relaunch the app (As I've enabled the 1s sync on launch), or would be lost if in the meantime the target was changed. 
Is it possible to push the local changes to the target as soon as they happen without waiting for the sync interval?

**Comment by fyears** [comment]
There is a "Schedule For Auto Run" setting in basic settings.

**Reply by idanangel** [reply]
Right, I have set the schedule for the shortest available period of 1 minute. My suggestion is to make local changes (edits, new files, deletes) pushed immediately to the target (e.g. Dropbox) instead of waiting for the scheduled sync, so that when I'm doing those changes on mobile and go to another app, the changes are not lost. Is it possible to change the plugin to push local changes to the target as soon as they happen instead of waiting for the auto-sync to kick start?

**Reply by jamesvclements** [reply]
+1

**Reply by jamesvclements** [reply]
and +1 on how amazing this plug-in is, couldn't believe how well it worked and how easy it was to setup @fyears

**Reply by erohtar** [reply]
+1 Please

**Comment by berot3** [comment]
I agree, shall we not create a feature request instead of leaving this here in disussions 🤔

**Comment by simonla** [comment]
+1
And this feature is so easy to implement

![image](https://user-images.githubusercontent.com/14934570/201401657-e8e199d7-c4d3-4a05-be5f-77c3b3089269.png)

**Comment by shanmukhuggini** [comment]
anyone working on google drive based one just like one drive ? I'm keen to give an hand

**Reply by fyears** [reply]
It is in my plan and expected to be implemented in a few months

**Reply by shanmukhuggini** [reply]
Can I contribute as well?

**Reply by fyears** [reply]
> Can I contribute as well?

Thanks but I would like to develop it by myself firstly because there are some big updates and refactors and I want to make sure everything works properly quickly. Fixes and bug reports are greatly appreciated after the function getting implemented and release! Thanks again!

**Reply by berkdemir** [reply]
Is there an eta for this?

**Reply by fyears** [reply]
google drive is implemented

**Comment by shanmukhuggini** [comment]
yes, but as pro feature. till what date it is free?

Thanks,
Sai U

**Reply by fyears** [reply]
till June 30
